press-sync.php

```diff
@@ -237,15 +244,15 @@ public function get_objects_to_sync( $objects_to_sync, $paged = 1, $taxonomies,
 	 * @since 0.1.0
 	 *
 	 * @param string $objects_to_sync
-	 * @param integer $paged
+	 * @param integer $next_page
 	 * @param array $taxonomies
 	 * @return array posts
 	 */
-	public function get_posts_to_sync( $objects_to_sync, $paged = 1, $taxonomies, $where_clause = '' ) {
+	public function get_posts_to_sync( $objects_to_sync, $next_page = 1, $taxonomies, $where_clause = '' ) {
 
 		global $wpdb;
 
-		$offset = ( $paged > 1 ) ? ( $paged - 1 ) * 5 : 0;
+		$offset = ( $next_page > 1 ) ? ( $next_page - 1 ) * 5 : 0;
 
 		$where_clause 	= ( $where_clause ) ? ' AND ' . $where_clause : '';
 		$sql 			= "SELECT * FROM $wpdb->posts WHERE post_type = %s AND post_status NOT IN ('auto-draft','trash') $where_clause ORDER BY post_date DESC LIMIT 5 OFFSET %d";
```

Comment:
It would be good to wrap variables in `{}` to make it easier to read and to clarify string vs. variable:

```
$sql = "SELECT * FROM {$wpdb->posts} ..."
```

Reply:
Thanks. This has been updated.